core/invoker/src/main/scala/whisk/core/containerpool/DockerContainer.scala

```diff
@@ -107,10 +108,18 @@ class DockerContainer(id: ContainerId, ip: ContainerIp)(
                 transid.finished(this, start.copy(start = r.interval.start), s"initialization result: ${r.toBriefString}", endTime = r.interval.end)
             case Failure(t) =>
                 transid.failed(this, start, s"initializiation failed with $t")
+        }.flatMap { result =>
```

Comment:
note to self: what is initializer itself fails - does that throw InitializationError also?